### ISSUE TITLE
forwardMessages promise never resolve or reject.

### ISSUE THREAD

**Issue by andRodFor** [orkestral/venom#1165]
Version tested: 3.1.4 and 3.1.6,
After executing the function, no callback or promise is received. Nothing happens.

Example code:
```
var to = "000000000@c.us";
var messageID = message.id;

forwardMessages(to, messageID, true).then((value: any) => {
            console.log("success?");
                }).catch((err: any) => {
               console.log(err);
                });
```

**Comment by paulocorcino** [comment]
For me, too forwardMessages not work, I stay waiting for a return.

**Comment by orkestral** [comment]
fixed in version 4.0.1

**Comment by riservato-thiagomelo** [comment]
> fixed in version 4.0.1

version 4.0.2 doesn't work
